prisma/schema.prisma

```diff
@@ -13,3 +13,17 @@ datasource db {
   url      = env("DATABASE_URL")
```

Comment:
**suggestion**: Consider renaming the `User` field in the `Link` model to `user` to follow the convention of using lowercase for field names. This can improve consistency and readability.